File: gastromanager-backend/src/main/java/com/kaiho/gastromanager/application/ingredient/dto/response/IngredientResponseDto.java
package com.kaiho.gastromanager.application.ingredient.dto.response;

import lombok.Builder;

import java.time.Instant;
import java.util.UUID;

@Builder
public record IngredientResponseDto(
        UUID uuid,
        String name,
        double availableStock,
        String unit,
        double pricePerUnit,
        String supplier,
        int minimumStockQuantity,
        String createdBy,
        Instant createdDate,
        String updatedBy,
        Instant updatedDate) {
}

File: gastromanager-frontend/src/app/core/model/interfaces/IngredientResponseDto.ts
export interface IngredientResponseDto {
  supplier: string;
  createdDate: string;
  createdBy: string;
  uuid: string;
  name: string;
  availableStock: number;
  unit: string;
  pricePerUnit: number;
  minimumStockQuantity: number;
  // updatedDate: string;
}
